src/stcal/alignment/resample_utils.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import numpy as np
+from util import wcs_bbox_from_shape, reproject
+from gwcs.wcstools import grid_from_bounding_box
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+
+def calc_pixmap(in_wcs, out_wcs, shape=None):
+    """Return a pixel grid map from input frame to output frame
+
+    Parameters
+    ----------
+    in_wcs: `~astropy.wcs.WCS`
+        Input WCS objects or transforms.
+    in_wcs: `~astropy.wcs.WCS`
+        output WCS objects or transforms.
+    shape : tuple, optional
+        Shape of grid in pixels. The default is None.
+
+    Returns
+    -------
+    pixmap
+        reprojected pixel grid map
+    """
+    if shape:
+        bb = wcs_bbox_from_shape(shape)
+        log.debug("Bounding box from data shape: {}".format(bb))
+    else:
+        bb = in_wcs.pixel_shape
```

Comment:
Does this return a shape tuple or a bounding_box? Should `in_wcs.bounding_box` be used? Or alternatively use the bounding_box and if it's not available create one from the shape.

Reply:
It doesn't look like bounding_box is available for fitswcs so I'll just use `wcs_bbox_from_shape(in_wcs.pixel_shape)` unless there is a better way. 

Thanks for the suggestions.